handel/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "extends": "../tsconfig",
   "compilerOptions": {
-    "allowJs": true
+    "allowJs": true,
+    "rootDir": "./"
```

Comment:
I looked at the tsconfig docs for this, but I'm still a little unclear on what it does. Can you tell me how this changes the compilation?

Reply:
Among other things, it disallows relative paths that leave that directory, which is good because my IDE kept wanting to import the extension api from relative paths instead of from the node package.